src/components/change-wallet/AddressRow.js

```diff
@@ -2,8 +2,8 @@ import React, { useCallback, useMemo } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import LinearGradient from 'react-native-linear-gradient';
 import { useTheme } from '../../context/ThemeContext';
-import { removeFirstEmojiFromString } from '../../helpers/emojiHandler';
 import { deviceUtils } from '../../utils';
+import { addressHashedEmoji } from '../../utils/defaultProfileUtils';
```

Comment:
you should export `addressHashedEmoji` from the root `utils` directory

and then consolidate this line with the line above

and then you should use the `@rainbow-me/` alias

so final result is : 
`import { addressHashedEmoji, deviceUtils } from '@rainbow-me/utils';`